cms/forms/utils.py

```diff
@@ -26,6 +26,8 @@ def get_sites():
 
 
 def get_page_choices_for_site(site, language):
+    from cms.models import PageContent
```

Comment:
why?

Reply:
I did this because when I imported `PlaceholderRelationField` from `cms.models.fields` in titlemodels.py, cms/forms/utils.py throws an ImportError on PageContent. That's why it was moved inside the function.